src/components/layout/Footer.tsx

```diff
@@ -35,6 +45,7 @@ const Footer: React.FC = () => (
         <Link to="/source-code">Source code</Link>
       </li>
     </ul>
+    <div className="version">dFusion PoC v{VERSION}</div>
```

Comment:
Newbie question: I see that you imported the file containing the `VERSION` variable on `App.tsx`, but not here. I assume it works. The question is: why it works?

Reply:
Hehe, good question :) 

If I'm honest, this was also new to me, I haven't used too much typescript in projects before.
What I think is, by using `declare` it becomes global. I just imported it in the App cause is the entry point (could have been imported in `index.ts` too).